src/iommu/iommufd.c

```diff
@@ -49,6 +50,7 @@ struct iommufd_device_info {
 
 	int dev_fd;
 	int dev_id;
+	int hwpt_id;
```

Comment:
Since we do not expose `struct iommufd_device_info` in the public headers, I see why we could get away with not storing it. We do not even need it for detaching.